sdk/python/v1beta1/setup.py

```diff
@@ -85,4 +85,7 @@
         "Topic :: Software Development :: Libraries :: Python Modules",
     ],
     install_requires=REQUIRES,
+    extras_require={
+        "huggingface": ["kubeflow-training[huggingface]"],
```

Comment:
Should we use the specific version of Training SDK here ?
```suggestion
        "huggingface": ["kubeflow-training[huggingface]==1.8.0"],
```

Reply:
Yeah, I agree it will be better to specify version here. I have updated accordingly.